docs/src/main/asciidoc/mutiny-primer.adoc

```diff
@@ -288,16 +288,40 @@ Fortunately, Mutiny provides a set of shortcut to make your code more concise:
 
 
 |===
-|Shortcut |Equivalent
-
-| uni.map(x → y) | uni.onItem().transform(x → y)
-| uni.flatMap(x → uni2) | uni.onItem().transformToUni(x → uni2)
-| uni.chain(x → uni2) | uni.onItem().transformToUni(x → uni2)
-| uni.then(() → uni2) | uni.onItem().transformToUni(ignored → uni2)
-| uni.invoke(x → System.out.println(x)) | uni.onItem().invoke(x → System.out.println(x))
-| uni.call(x → uni2) | uni.onItem().call(x → uni2)
-| uni.eventually(() → System.out.println("eventually")) | uni.onItemOrFailure().invoke((ignoredItem, ignoredException) → System.out.println("eventually"))
-| uni.eventually(() → uni2) | uni.onItemOrFailure().call((ignoredItem, ignoredException) → uni2)
+|Shortcut | Equivalent
+
+|`uni.map(x -> y)`
+|`uni.onItem().transform(x -> y)`
+
+|`uni.flatMap(x -> uni2)`
+|`uni.onItem().transformToUni(x -> uni2)`
+
+|`uni.chain(x -> uni2)`
+|`uni.onItem().transformToUni(x -> uni2)`
+
+|`uni.then(() -> uni2)`
+|`uni.onItem().transformToUni(ignored -> uni2)`
+
+|`uni.invoke(x -> System.out.println(x))`
+|`uni.onItem().invoke(x -> System.out.println(x))`
+
+|`uni.call(x -> uni2)`
+|`uni.onItem().call(x -> uni2)`
+
+|`uni.eventually\(() -> System.out.println("eventually"))`
+|`uni.onItemOrFailure().invoke\((ignoredItem, ignoredException) -> System.out.println("eventually"))`
+
+|`uni.eventually(() -> uni2)`
+|`uni.onItemOrFailure().call((ignoredItem, ignoredException) -> uni2)`
```

Comment:
Why do you have a `\` for `((` except here?
Does it have a special meaning in Asciidoc?

Reply:
I remember having the same problem in the Mutiny doc, for some reason sometines rendering is broken with `((`:

![Screenshot 2022-06-22 at 14 27 56](https://user-images.githubusercontent.com/25961/175028889-cecb4a8c-92fc-4b64-82a1-517f3b289f4f.png)

Reply:
OK maybe try to add the escaping character to this one too as my guess is that it might break if somehow you add a `))` after that line.

Reply:
Well it renders fine, don't ask me why 🤣 

![Screenshot 2022-06-22 at 14 36 46](https://user-images.githubusercontent.com/25961/175030548-a37d8d5f-16ab-4574-9b25-68286baf3fe7.png)

Reply:
And it still renders fine if you add a `))` on a line below that one?

Reply:
They get rendered as `\((` instead of `((`